examples/simple_zonal_with_hub/main.tf

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
+  cluster_type = "simple-zonal"
```

Comment:
I think rather than having an example just for hub registration, I think it makes sense to fold this into the ASM example? This will allow us to leverage the integration tests 
https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/blob/8a9f9041c8f18ff7a873873e9b19e03dcdfe7d2a/test/integration/simple_zonal_with_asm/controls/gcloud.rb#L54
already written that checks for hub membership. WDYT?

Reply:
Thoughts about integrating in ACM instead?

Reply:
Yup thats fine too, you can use that same tests written for ASM fixture and port them to ACM.

Reply:
Actually, thinking through this, having an example focused on just the module in question is useful, but I also see the value in the integration. Suggest we leave this as is and open a new PR to remove hub from asm and at the same time create an Anthos example with all three modules (hub, acm, asm) combined. This example can be the integration test and include any future modules.  Thoughts?

Reply:
Sorry to wade in but this is exactly what I do now vpc, cluster, asm, acm (for the time being I am dropping in upstream Istio in favor of ASM version - but mix and match) so having the hub split out would be excellent for my use case.

Reply:
@cgrant that sgtm. We can address the changes to ASM in a separate PR. I also like the idea of an Anthos example 🚀